waku/node/peer_manager/peer_store/waku_peer_storage.nim

```diff
@@ -21,29 +21,41 @@ export db_sqlite
 type
   WakuPeerStorage* = ref object of PeerStorage
     database*: SqliteDatabase
-    replaceStmt: SqliteStmt[(seq[byte], seq[byte], int32, int64), void]
+    replaceStmt: SqliteStmt[(seq[byte], seq[byte]), void]
 
 ##########################
 # Protobuf Serialisation #
 ##########################
 
-proc init*(T: type RemotePeerInfo, buffer: seq[byte]): ProtoResult[T] =
+proc decode*(T: type RemotePeerInfo, buffer: seq[byte]): ProtoResult[T] =
   var
     multiaddrSeq: seq[MultiAddress]
     protoSeq: seq[string]
     storedInfo = RemotePeerInfo()
+    rlpBytes: seq[byte]
+    connectedness: uint32
+    disconnectTime: uint64
 
   var pb = initProtoBuffer(buffer)
 
   discard ? pb.getField(1, storedInfo.peerId)
   discard ? pb.getRepeatedField(2, multiaddrSeq)
   discard ? pb.getRepeatedField(3, protoSeq)
   discard ? pb.getField(4, storedInfo.publicKey)
-
-  # TODO: Store the rest of parameters such as connectedness and disconnectTime
+  discard ? pb.getField(5, connectedness)
+  discard ? pb.getField(6, disconnectTime)
+  let hasENR = ? pb.getField(7, rlpBytes)
```

Comment:
Are we testing the case where the peer doesn't have ENR info? I wonder if that case may make it crash at this point.

Reply:
:thinking: Would be good to know indeed.